packages/headless/src/ssr.index.ts

```diff
@@ -10,12 +10,12 @@ export type {AnalyticsClientSendEventHook} from 'coveo.analytics';
 // Main App
 // ⚠️ NOTE: All exported SSR types, APIs should be marked as `@internal` until MVP is complete
 export type {
-  SearchEngine,
   SearchEngineOptions,
   SearchEngineConfiguration,
   SearchConfigurationOptions,
 } from './app/search-engine/search-engine';
 export type {
+  SSRSearchEngine as SearchEngine,
```

Comment:
Internally I named the new function `SSRSearchEngine` to differentiate it, since `SearchEngine` already exists.

From an implementers' perspective, when they are loading the search engine from `@coveo/headless/ssr`, they are already importing from "SSR", so it can't be anything else than the SSR search engine. Therefore, I felt the prefix was redundant (and ugly) so I removed it there.

I can change it if you prefer 🤷

Reply:
Makes sense 👍